html/datasets.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Can we add a comment here linking to the favetview2 github repository? We need to give it proper citation here. Also, it's a place where we can read about the different capabilities of the tool, etc...

Something like:

This web page utilizes the functionality of the facetview2 (https://github.com/CottageLabs/facetview2) faceted interface to elasticsearch (https://www.elastic.co/products/elasticsearch).